dotnet/src/SemanticKernel/Planning/Plan.cs

```diff
@@ -256,7 +256,7 @@ public async Task<Plan> InvokeNextStepAsync(SKContext context, CancellationToken
 
             var result = await step.InvokeAsync(functionContext, cancellationToken: cancellationToken).ConfigureAwait(false);
 
-            var resultValue = result.Result.Trim();
+            var resultValue = result.Context.Result.Trim();
```

Comment:
I'm wondering if this should be result.Value instead.

Reply:
Good catch! I made it `result.GetValue<string>() ?? result.Context.Result` just for backward compatibility, but I tested manually, and `result.GetValue<string>()` works as expected. We can remove `result.Context.Result` in the future.